deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```
@@ -228,7 +228,7 @@ processors:
       - type: json_parser
         parse_from: body
         parse_to: body
-        if: body matches "^{.*"
+        if: body matches "^{.*}"
```

alternatively it can be:

```suggestion
        if: body matches "^{.+"
```

---

```suggestion
        if: body matches "^{[\\s\\S]+"
```